README.md

```diff
@@ -1,3 +1,7 @@
 # ExaAM Workflows
 
 A number of workflows may exist for additive manufacturing processes. As our collections of workflows grow, the ExaAM project will provide examples of workflows here. Currently, we provide our uncertainty quantification workflow to model a location within the NIST AMBench-2018-01 part on Frontier. You can find this workflow in the `uq_workflow` directory along with directions related to how to run it.
+
```

Comment:
So not everyone is going to be able to run docker images on the HPC system they might be on and that would be the only reason I'd be hesitant to remove the AdditiveFoam submodule. However, I think it would be okay to remove that submodule if we can easily point to them the spack packages and be like this is another sane way of doing it if you can't use docker.

Reply:
I'm not entirely sure the limits of docker on HPC, but my initial intent was to have something that could be run on a local system. As for the submodule, I'll be honest I'm not very familiar and I didn't understand how it would be used - wouldn't it still require a full OpenFOAM build somewhere and wouldn't we also want submodules for all codes in that case?

Reply:
So, I had added the submodule just as AdditiveFoam had to be built locally to be used to be run, while the other codes could be built elsewhere on the system. So, the submodule allowed us to avoid having to include the repo directly here. However, it sounds like with the spack build of AdditiveFoam that's no longer the case? If so I'm okay telling people to use spack to build at least AdditiveFoam on a HPC system / local system for things as well if they can't make use of a docker image.